Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,47 @@ class Program
     {
         static void Main(string[] args)
         {
+            Lesson3.Demo();
+        }
+
+        public static void Descriptor()
+        {
+            //ask user for their name, age, weight, and height
+            string name = PromptString("Name: ");
+            string surname = PromptString("Surname: ");
+            int age = PromptInt("Age: ");
+            float weight = PromptFloat("Weight: ");
+            float height = PromptFloat("Height: ");
+            
+            //print a sentence about user
+            Console.WriteLine($"{name} {surname} is {age} years old, their weight is {weight}Kg and their height is {height}cm");
+
+            //calculate and print the BMI of the user
+            float bmi = CalculateBMI(weight, height / 100);
+            Console.WriteLine($"They have a BMI of {bmi}");
+        }
+
+        public static int PromptInt(string prompt)
+        {
+            Console.Write(prompt);
+            return int.Parse(Console.ReadLine());
+        }
+
+        public static string PromptString(string prompt)
+        {
+            Console.Write(prompt);
+            return Console.ReadLine();
+        }
+
+        public static float PromptFloat(string prompt)
+        {
+            Console.Write(prompt);
+            return float.Parse(Console.ReadLine());
+        }
+
+        public static float CalculateBMI(float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647)